integration/derp_verify_endpoint_test.go

```diff
@@ -76,21 +80,40 @@ func TestDERPVerifyEndpoint(t *testing.T) {
 	allClients, err := scenario.ListTailscaleClients()
 	assertNoErrListClients(t, err)
 
+	fakeKey := key.NewNode()
+	DERPVerify(t, fakeKey, derpRegion, false)
```

Comment:
I think this is good, main thing is to validate that nodes that should not have access dont have.